### ISSUE TITLE
Change font because 1 and l are similar.

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/styles#168]
@dakane1 opened https://github.com/swcarpentry/shell-novice/issues/612 saying that

> I had a number of issues distinguishing 1 (ones) from the letter l. When I mentioned it to a few people they said they were having the same problem.

# Screenshot

![screenshot from 2017-08-31 19-29-21](https://user-images.githubusercontent.com/1506457/29939195-fd4c6740-8e82-11e7-9472-5a130e2a8cc5.png)

# Technical details

If you are using a high resolution display (HIDPI) like the retina display in some MacBooks, the QHD+ in some laptops or (you are the luck one with) the 4k monitor you may have some issues due many factors that some fonts doesn't scale properly and the text looks **super** small making hard to read and distinguish some symbols that are similar. **If this is your case, sorry but we can't do much for you.**

The CSS related information about the font that we are using in the main body of the lesson is

~~~
font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
font-size: 14px;
line-height: 1.42857143;
color: #333;
~~~

Because of my completely lack of design skills, I'm open to suggestions about what we should use as `font-family`. **As long as you suggest a open source font**, I will not oppose to make the change.

**Comment by dakane1** [comment]
I had actually printed the lessons and was not reading them on a screen, not sure if that information helps or not.  http://swcarpentry.github.io/shell-novice/aio/.  I can ask a web designer I know to see if they have any font recommendations.

**Comment by rgaiacs** [comment]
@dakane1 Thanks for the information. It is very useful.

For the main text, the only difference of the CSS used for print is

~~~
color: #000 !important;
~~~

which enforce a darker colour.

> I can ask a web designer I know to see if they have any font recommendations.

Thanks in advance.

**Comment by rgaiacs** [comment]
@dakane1 did you have the opportunity to ask for suggestion from your web designer friend?

**Comment by dakane1** [comment]
Sorry, not yet.  Start of the quarter has been crazy busy but I will ask after OA Week, which is next week.

**Comment by fmichonneau** [comment]
Here are some options:

# Cabin

License: Open Font License

`1Il0O`: 
![screenshot from 2018-06-15 09-50-34](https://user-images.githubusercontent.com/5502922/41471417-950b89ea-7081-11e8-8014-9731a753ece1.png)

Preview:
![screenshot from 2018-06-15 09-49-37](https://user-images.githubusercontent.com/5502922/41471425-9ea4435c-7081-11e8-813d-d58ba3f76b9f.png)

# Open Sans

License: Apache License 2.0

`1Il0O`:
![screenshot from 2018-06-15 09-53-09](https://user-images.githubusercontent.com/5502922/41471550-f60f7a12-7081-11e8-9f15-4e8516c2c4a5.png)

Preview:
![screenshot from 2018-06-15 09-51-58](https://user-images.githubusercontent.com/5502922/41471555-fbe2b0d0-7081-11e8-8508-4b5c81251c76.png)

# Source Sans Pro

License: Open Font License

`1Il0O`: 
![screenshot from 2018-06-15 09-55-28](https://user-images.githubusercontent.com/5502922/41471676-4f5e2802-7082-11e8-84e4-85ca830448f0.png)

Preview:
![screenshot from 2018-06-15 09-56-27](https://user-images.githubusercontent.com/5502922/41471722-68e77dc8-7082-11e8-9fd3-f4702b1e5937.png)

# Ubuntu

License: [Ubuntu Font License](http://font.ubuntu.com/ufl/)

`1Il0O`: 
![screenshot from 2018-06-15 09-57-45](https://user-images.githubusercontent.com/5502922/41471880-ca240c00-7082-11e8-822f-4bab339772a6.png)

Preview:
![screenshot from 2018-06-15 09-58-30](https://user-images.githubusercontent.com/5502922/41471891-d0ab3e0e-7082-11e8-9ae3-e2c7f8565ead.png)

**Comment by maxim-belkin** [comment]
I like the last two (**Source Sans Pro** and **Ubuntu**) the most:
- In **Cabin**: zero and capital "o" look similar
- In **Open Sans**: "el" looks similar to capital i

What font do code blocks in the "Ubuntu" example use? I see that number one in the `wc -l *.pdb | sort -n | head -n 1` looks different from how it looks in the title

**Comment by fmichonneau** [comment]
Another option

# Abeezee

`1lI0O`:
![screenshot from 2018-06-15 10-35-27](https://user-images.githubusercontent.com/5502922/41473637-de8b38e4-7087-11e8-88b1-4daa8c4fc016.png)

Preview:

![screenshot from 2018-06-15 10-37-29](https://user-images.githubusercontent.com/5502922/41473739-246907e2-7088-11e8-8c82-f2619638046f.png)

**Comment by fmichonneau** [comment]
@maxim-belkin I don't think the font for the code was changed across these examples. It looks like on my laptop, it's rendered as Consolas.

**Comment by maxim-belkin** [comment]
That's a nice one too.

**Comment by dakane1** [comment]
I agree with Maxim; Ubuntu, Source Sans Pro, and Abeezee all look like they may work well since you can distinguish between zero and O plus 1 and l.

--Danielle

Danielle Kane
Research Librarian for Emerging Technologies and Service Innovation
Interim Data Management & Curation Librarian
LAUC-I Chair
University of California, Irvine Libraries
949.824.2024
Skype: da.kaneuci
[zotsmarter-logo]

**Comment by brownsarahm** [comment]
I think even Ubuntu and Abeeze are slightly better on the zero versus O than the source sans pro. and Abeeze is slightly more distinct on the letters `l` vs `I`

**Comment by hoytpr** [comment]
Ubuntu or Abeezee look good. I agree with @brownsarahm 's comments.

**Comment by maxim-belkin** [comment]
A friend of mine with dyslexia says Ubuntu is the most readable one for him.
